VSDS-LDESServer4J-infra-mongo/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/LdesFragmentStorageService.java

```diff
@@ -0,0 +1,21 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.services.LdesFragmentRepository;
+import be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.entity.LdesFragmentEntity;
+import be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.repository.LdesFragmentMongoRepository;
+import lombok.RequiredArgsConstructor;
+import org.json.simple.JSONObject;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class LdesFragmentStorageService implements LdesFragmentRepository {
```

Comment:
This is wrong.
LdesFragmentStorageServiceDefault implements LdesFragmentStorageService and has a private final member LdesFragmentRepository (unimplemented interface -> Spring Boot takes care of it).